### PR TITLE
Adding cmyt

### PR DESCRIPTION
**Pull request by neutrinoceros**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [N/A] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by matthewturk** [comment]
Ι am willing to be listed.

File: recipes/cmyt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cmyt-{{ version }}.tar.gz
+  sha256: 9090d477cb9e11c1067887489112ffc6fb5fbaaa994dd111167c04d90dd51e34
+
+build:
+  number: 0
+  skip: true   # [py<36]
```
Comment (dopplershift) [inline recipes/cmyt/meta.yaml:15]: This looks like it's pure Python and could be noarch? If so, add this here and adjust the dependence on `typing-extensions` below:
```suggestion
  noarch: python
```

File: recipes/cmyt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (dopplershift) [inline recipes/cmyt/meta.yaml:21]: For noarch:
```suggestion
    - python >=3.6
```

File: recipes/cmyt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  run:
+    - colorspacious >=1.1.2
+    - matplotlib-base >=2.1.0
+    - more-itertools >=8.4
+    - numpy >=1.13.3
+    - python
+    - typing-extensions >=3.10.0.2  # [py<38]
```
Comment (dopplershift) [inline recipes/cmyt/meta.yaml:28]: For noarch:
```suggestion
    - python >=3.6
    - typing-extensions >=3.10.0.2
```

**Review by dopplershift — COMMENTED** [review]
In general, we try to be noarch as much as possible. In this case, it would be much better to just unconditionally depend on `typing-extensions`.

**Comment by neutrinoceros** [comment]
@dopplershift thank you for your review !

**Comment by mwcraig** [comment]
Thanks for the contribution @neutrinoceros!

**Pull request merged**